packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -36,24 +37,50 @@ const MIN_GAS_PRICE = 3; // Safelow gas price from GasStation, in Gwei
 @withEthBalance // ETH balance
 @observer
 class Send extends Component {
+  isCancelled = false;
+
   state = {
-    maxSelected: false
+    estimatedTxFee: ZERO,
```

Comment:
Remove this from state. The estimated fee is final-form's `values.gas` time `values.gasPrice`, so no need for another state variable.